src/uniparc/components/entry/XRefsFacets.tsx

```diff
@@ -1,123 +1,155 @@
 import { useMemo, FC, ReactNode } from 'react';
 import { countBy, uniqBy } from 'lodash-es';
-import { Facets, Facet } from 'franklin-sites';
+import { Facets, Loader } from 'franklin-sites';
 
 import { EntryTypeIcon } from '../../../shared/components/entry/EntryTypeIcon';
 
 import { EntryType } from '../../../uniprotkb/adapters/uniProtkbConverter';
 
 import {
-  UniParcXRef,
   databaseToEntryType,
+  UniParcAPIModel,
+  UniParcXRef,
+  XRefsInternalDatabases,
 } from '../../adapters/uniParcConverter';
-import { FacetValue } from '../../../uniprotkb/types/responseTypes';
+import {
+  FacetObject,
+  FacetValue,
+} from '../../../uniprotkb/types/responseTypes';
 import { OrganismData } from '../../../uniprotkb/adapters/namesAndTaxonomyConverter';
+import { UseDataAPIWithStaleState } from '../../../shared/hooks/useDataApiWithStale';
+
+import '../../../shared/components/results/styles/results-view.scss';
 
-const sortByCount = (a: FacetValue, b: FacetValue) => b.count - a.count;
+const reviewed: XRefsInternalDatabases = 'UniProtKB/Swiss-Prot';
+const unreviewed: XRefsInternalDatabases = 'UniProtKB/TrEMBL';
+const exceptions = new Set<string>([reviewed, unreviewed]);
```

Comment:
Could be `new Set< XRefsInternalDatabases>` as this is what the set strictly consists of and then using `a.value as XRefsInternalDatabases` when checking membership

Reply:
Same comment than for the map. I need this to accept generic strings, not just those 2. I changed with a union just to be clear what it contains